metadata-io/build.gradle

```diff
@@ -18,7 +18,8 @@ dependencies {
   compile spec.product.pegasus.generator
 
   compile externalDependency.dgraph4j exclude group: 'com.google.guava', module: 'guava'
-  compile externalDependency.lombok
+  compileOnly externalDependency.lombok
+  implementation externalDependency.commonsCollections
```

Comment:
is this serving as replacement for velocity?

Reply:
I think maybe we were utilizing a transitive dependency from velocity on Apache Commons Collections. This just covers the previous build failure where CollectionUtils was not found.